src/plugins/workspace/public/services/use_case_service.ts

```diff
@@ -65,24 +65,32 @@ export class UseCaseService {
         if (navGroupInfo) {
           setupDeps.chrome.navGroup.addNavLinksToGroup(navGroupInfo, [
             {
-              id: 'dataSources',
+              id: 'objects',
               category: DEFAULT_APP_CATEGORIES.manageWorkspace,
               order: 100,
             },
             {
-              id: 'indexPatterns',
+              id: 'dataSources',
               category: DEFAULT_APP_CATEGORIES.manageWorkspace,
               order: 200,
             },
             {
-              id: 'objects',
+              id: 'indexPatterns',
               category: DEFAULT_APP_CATEGORIES.manageWorkspace,
               order: 300,
             },
             {
-              id: WORKSPACE_DETAIL_APP_ID,
+              id: 'import_sample_data',
```

Comment:
Do we still have sample data card in overview pages?

Reply:
Yes, but we still need the menu in left nav or the bread crumb info will be broken.

Reply:
I am having a hard time finding the Add Sample Data card in the workspaces other than Search. Is this something we added in the past day or two?